ui/src/components/auth/RegisterContainer.tsx

```diff
@@ -208,7 +214,7 @@ const RegisterContainer: React.FC = () => {
                 variant="button"
                 textAlign={'center'}
               >
-                Already have an account? Login
+                <Typography variant="subtitle1">Already have an account? Login</Typography>
```

Comment:
Can we split the "Already have an account?" and "Login"? so that Login can be a different color too :D